src/Dapr.Actors/Client/ActorProxy.cs

```diff
@@ -33,6 +38,11 @@ protected internal ActorProxy()
         {
         }
 
+        /// <summary>
+        /// The <see cref="JsonSerializerOptions"/> used for Actor messaging
+        /// </summary>
+        public JsonSerializerOptions JsonSerializerOptions { get; set; } = JsonSerializerDefaults.Web;
```

Comment:
Does this need to be public and get/set? The user already has a way to control this. It's little confusing that defaults are applied at multiple levels. Without enabling some additional scenario I think it would be better to treat this as an implementation detail of the class.

Reply:
Currently the only way a user would be able to control the serialization options of the proxy is by setting them in the ActorProxyOptions on the DefaultProxyFactory. Once they get an instance of the ActorProxy the won't be able to change the serialization options on a per instance scale.

That being said - I added the `ActorProxyOptions` as an option parameter to the static `Create` methods to give the users the ability to set the options on a per proxy basis. I think this was the intention from the start that I had overlooked. I also made the property JsonSerializerOptions on `ActorProxy` into a private field.